### PR TITLE
Fix Shopfa

### PR DESCRIPTION
**Pull request by Aghaie**

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -8469,6 +8469,19 @@
...
     },
+    "Shopfa": {
+      "cats": [
+        "6"
+      ],
+     "js": {
+        "shopfa": ""
+      },
+      "meta": {
+        "generator": "^ShopFA"
```
Comment (jvoisin) [inline src/apps.json]: Please add version detection too :)

Comment (Aghaie) [reply src/apps.json]: Thanks, Version Added.

**Comment by gadcam** [comment]
I think you can also add this pattern:
```json
"html": "<!--[^>]+Cooked by ShopFA v([\\d.]+)\\;version:\\1"
```
Are you sure the js pattern is working ? I could not find a website where it would match.

File: src/apps.json (hunk 1 above)
Comment (Aghaie) [reply src/apps.json]: We use this meta tag in Shopfa sites:

`<meta name="generator" content="ShopFA 8.1.786" />`

is this format Correct? 

```
      "meta": {
        "generator": "^ShopFA (.+)$\\;version:\\1"
      },
```

**Comment by Aghaie** [comment]
We use this meta tag in Shopfa sites:

`<meta name="generator" content="ShopFA 8.1.786" />
`
is this format Correct?

```
      "meta": {
        "generator": "^ShopFA (.+)$\\;version:\\1"
      }
```

**Comment by gadcam** [comment]
Yes it looks perfect :)

Ps: You could be a bit more accurate by changing <code>(.+)</code> into <code>([\\\\d.]+)</code>, it is maybe a punctilious precision however.

**Pull request closed without merging**